pkgs/applications/editors/vim/plugins/neovim-require-check-hook.sh

```diff
@@ -10,7 +10,7 @@ neovimRequireCheckHook () {
 		# editorconfig-checker-disable
         export HOME="$TMPDIR"
         @nvimBinary@ -es --headless -n -u NONE -i NONE --clean -V1 \
-            --cmd "set rtp+=$out,${dependencies/ /,}" \
+            --cmd "set rtp+=$out,${dependencies// /,}" \
```

Comment:
This patch should definitely get its own PR.
cc @teto

Reply:
Also, I am afraid that it will need to target staging as the number of rebuilds is significant.

Reply:
I will extract into separate pr

Reply:
https://github.com/NixOS/nixpkgs/pull/342240
will disable the check hook while the fix is not merged